gfootball/env/remote_football_env.py

```diff
@@ -90,6 +91,8 @@ def action_space(self):
   def step(self, action):
     if self._game_id is None:
       raise RuntimeError('Environment should be reset!')
+    if np.isscalar(action):
+      action = int(action)
     if isinstance(action, int):
```

Comment:
Isn't it sufficient to change this line to 
np.isscalar(action):
?

Reply:
Definitely! I'd still typecast the action to an int since I don't know how it's being handled on the server side.